### PR TITLE
Fix 'tethys_quotas.UserQuota' has no ForeignKey error

### LINKED ISSUE #777: tethys_quotas.UserQuota' has no ForeignKey to 'tethys_apps.ProxyApp
I have been occationally been getting this error. It seems like restarting the server resolves it, but I'm not sure why.

<details>
Environment:

Request Method: GET
Request URL: http://vtime.erdc.dren.mil/admin/tethys_apps/proxyapp/6/change/

Django Version: 2.2.14
Python Version: 3.7.12
Installed Applications:
('django.contrib.admin',
 'django.contrib.auth',
 'django.contrib.contenttypes',
 'django.contrib.sessions',
 'django.contrib.messages',
 'django.contrib.staticfiles',
 'django_gravatar',
 'bootstrap3',
 'termsandconditions',
 'tethys_config',
 'tethys_apps',
 'tethys_gizmos',
 'tethys_services',
 'tethys_compute',
 'tethys_quotas',
 'social_django',
 'guardian',
 'session_security',
 'captcha',
 'snowpenguin.django.recaptcha2',
 'rest_framework',
 'rest_framework.authtoken',
 'analytical',
 'channels',
 'mfa',
 'axes')
Installed Middleware:
('django.contrib.sessions.middleware.SessionMiddleware',
 'django.middleware.common.CommonMiddleware',
 'django.middleware.csrf.CsrfViewMiddleware',
 'django.contrib.auth.middleware.AuthenticationMiddleware',
 'django.contrib.messages.middleware.MessageMiddleware',
 'tethys_portal.middleware.TethysMfaRequiredMiddleware',
 'django.middleware.clickjacking.XFrameOptionsMiddleware',
 'tethys_portal.middleware.TethysSocialAuthExceptionMiddleware',
 'tethys_portal.middleware.TethysAppAccessMiddleware',
 'session_security.middleware.SessionSecurityMiddleware',
 'axes.middleware.AxesMiddleware')

Traceback:

File "./django/core/handlers/exception.py" in inner
  34.             response = get_response(request)

File "./django/core/handlers/base.py" in _get_response
  115.                 response = self.process_exception_by_middleware(e, request)

File "./django/core/handlers/base.py" in _get_response
  113.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "./django/contrib/admin/options.py" in wrapper
  606.                 return self.admin_site.admin_view(view)(*args, **kwargs)

File "./django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "./django/views/decorators/cache.py" in _wrapped_view_func
  44.         response = view_func(request, *args, **kwargs)

File "./django/contrib/admin/sites.py" in inner
  223.             return view(request, *args, **kwargs)

File "./django/contrib/admin/options.py" in change_view
  1648.         return self.changeform_view(request, object_id, form_url, extra_context)

File "./django/utils/decorators.py" in _wrapper
  45.         return bound_method(*args, **kwargs)

File "./django/utils/decorators.py" in _wrapped_view
  142.                     response = view_func(request, *args, **kwargs)

File "./django/contrib/admin/options.py" in changeform_view
  1529.             return self._changeform_view(request, object_id, form_url, extra_context)

File "./django/contrib/admin/options.py" in _changeform_view
  1590.                 formsets, inline_instances = self._create_formsets(request, obj, change=True)

File "./django/contrib/admin/options.py" in _create_formsets
  1945.         for FormSet, inline in self.get_formsets_with_inlines(*get_formsets_args):

File "./django/contrib/admin/options.py" in get_formsets_with_inlines
  794.             yield inline.get_formset(request, obj), inline

File "./django/contrib/admin/options.py" in get_formset
  2131.         return inlineformset_factory(self.parent_model, self.model, **defaults)

File "./django/forms/models.py" in inlineformset_factory
  1053.     fk = _get_foreign_key(parent_model, model, fk_name=fk_name)

File "./django/forms/models.py" in _get_foreign_key
  1027.                     parent_model._meta.label,

Exception Type: ValueError at /admin/tethys_apps/proxyapp/6/change/
Exception Value: 'tethys_quotas.UserQuota' has no ForeignKey to 'tethys_apps.ProxyApp'.
</details>

### PR DESCRIPTION
**Pull request by sdc50**
fix 'tethys_quotas.UserQuota' has no ForeignKey error when adding a user through the admin pages
fixes #777

### CODE REVIEW

**Comment by sdc50** [comment]
@swainn Please review this carefully. This PR is just removing code. I don't understand why the code was needed in the first place, so I'm not sure if it will end up breaking something else.

File: tethys_apps/admin.py [hunk 1]
```diff
@@ -149,18 +149,6 @@ def has_add_permission(self, request, obj=None):
...
-    def add_view(self, *args, **kwargs):
-        if UserQuotasSettingInline in self.inlines:
-            self.inlines.pop(self.inlines.index(UserQuotasSettingInline))
-        return super().add_view(*args, **kwargs)
-
-    def change_view(self, *args, **kwargs):
-        if UserQuotasSettingInline not in self.inlines:
-            self.inlines.append(UserQuotasSettingInline)
-        return super().change_view(*args, **kwargs)
-
```
Comment (swainn) [inline tethys_apps/admin.py:162]: This defines a custom admin view for the User model and adds the field for controlling quotas on a per-user basis:

image

Comment (swainn) [reply tethys_apps/admin.py:162]: Removing this code would mean removing this feature. I'll take a look at it to see if I can figure out what is going on.

**Comment by sdc50** [comment]
@swainn okay, I think this will fix the issue and still keep the UseQuota setting in the User change_view.

**Review by swainn — APPROVED** [review]

**Pull request merged**